src/useCases/utils.ts

```diff
@@ -1,12 +1,14 @@
-import * as b from 'fp-ts/boolean';
+import * as E from 'fp-ts/lib/Either';
```

Comment:
Maybe we can rename this file to `authorize.ts` and move it under `domain` layer